src/components/TodoError.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+
+interface Props {
+  error: string | null;
+  setError: (arg0: string | null) => void,
+}
+
+export const TodoError: React.FC<Props> = ({ error, setError }) => {
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+
+  useEffect(() => {
+    switch (error) {
+      case 'cantfetch':
```

Comment:
Let's create an enum for that. You can directly set the error messages to for example ` Unable to fetch todos` instead of setting it based on the type